.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: tests
+on: [pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5.7, 2.6.5, 2.7.0]
+    env:
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      RAILS_ENV: test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
```

Comment:
Should add:

`bundler-cache: true # runs 'bundle install' and caches installed gems automatically`